examples/classification/README.md

```diff
@@ -34,3 +37,83 @@ python classification.py mobilenet_v2 --batch_size 8 --num_cores 4
+
+Example command:
+```bash
+python server.py "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate"
+```
+
+You can leave that running as a detached process or in a spare terminal.
+
+This starts a Flask app with the DeepSparse Engine as the inference backend, accessible at http://0.0.0.0:5543 by default.
+
+The app exposes HTTP endpoints at:
+
+- `/info` to get information about the compiled model
+- `/predict` to send inputs to the model and receive a response. The number of inputs should match the compiled model's batch size.
+For a full list of options, run `python server.py -h`.
+  
+### Client
+
+`client.py` provides a `Client` object to make requests to the 
+server easy.
+The file is self-documented.  See example usage below:
+
+```python
+from client import Client
+from helpers import load_data, _BatchLoader
+
+myclient = Client()
+resnet_stub = "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate"
+batch_loader = _BatchLoader(data=load_data(resnet_stub), batch_size=1, iterations=1)
+
+for batch in batch_loader:
+    out = myclient.detect(images=batch,)
+    print(f"outputs:{out}")
+```
+
+### SparseZoo Stubs
+[SparseZoo](http://sparsezoo.neuralmagic.com/) is a constantly-growing repository of sparsified (pruned and 
+pruned-quantized) models with matching sparsification recipes for neural networks. It simplifies and accelerates your time-to-value in building performant deep learning models with a collection of inference-optimized models and recipes to prototype from.
+
+Available via API and hosted in the cloud, the [SparseZoo](http://sparsezoo.neuralmagic.com/) contains both 
+baseline and models sparsified to different degrees of inference performance vs. baseline loss recovery. Recipe-driven approaches built around sparsification algorithms allow you to take the models as given, transfer-learn from the models onto private datasets, or transfer the recipes to your architectures.
+
+Each model in the SparseZoo has a specific stub that identifies it. The stubs are made up of the following structure:
+
+`DOMAIN/SUB_DOMAIN/ARCHITECTURE{-SUB_ARCHITECTURE}/FRAMEWORK/REPO/DATASET{-TRAINING_SCHEME}/SPARSE_NAME-SPARSE_CATEGORY-{SPARSE_TARGET}`
+
+Learn more at 
+- [SparseZoo Docs](https://docs.neuralmagic.com/sparsezoo/)
+- [SparseZoo Website](https://sparsezoo.neuralmagic.com/) 
+- [SparseZoo GitHub Repository](https://github.com/neuralmagic/sparsezoo)
+
+A few classification models from [SparseZoo](https://sparsezoo.neuralmagic.com/):
+
+| Model Name     |      Stub      | Description |
+|----------|-------------|-------------|
+| resnet-pruned-moderate | zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate |This model is a sparse, [ResNet-50](https://arxiv.org/abs/1512.03385) model that achieves 99% of the accuracy of the original baseline model (76.1% top1). Pruned layers achieve 88% sparsity.|
```

Comment:
let's also add the pruned_quantized resnet model and a pruned and pruned_quantized mobilenet

Reply:
sure!